nuage_metroae_config/user_data_parser.py

```diff
@@ -60,7 +60,10 @@ def _parse_data_string(self, user_data_string, filename="(internal)"):
         data_dict = self._decode_to_dict(user_data_string, filename)
         reader = Reader(self)
         reader.set_filename(filename)
-        reader.read(data_dict)
+        if data_dict is not None:
+            reader.read(data_dict)
+        else:
+            print "No Userdata content found in file: " + filename + " skipping"
```

Comment:
Please don't print anything.  There is a logging system, but I don't think it is necessary to even do that.

Reply:
Shouldn't we indicate to the user that the file was skipped?

Reply:
I don't think it is necessary.  Especially for the use-case Ben is providing.  It would be pretty annoying having 50 example files all being skipped in the output.

Reply:
Sounds good